lib/Documentable/Primary.pm6

```diff
@@ -130,12 +130,12 @@ class Documentable::Primary is Documentable {
             %attr = name       => $name.trim,
                     kind       => Kind::Syntax,
                     subkinds   => @meta || (),
-                    categories => @meta || ();
+                    categories => @(@meta[0]) || ();
```

Comment:
OK, before subkinds and categories could be the same. Now, subkinds can be a superset of categories?